src/Microsoft.Azure.SignalR.SignalRService/ServiceManagerBuilder.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.SignalR.SignalRService
+{
+    public class ServiceManagerBuilder
+    {
+        public ServiceManagerBuilder WithCredential(string conenctionString)
```

Comment:
I think "credentials" is more commonly used.

Reply:
How about `Authenticate`. `credentials` is a noun. I think `WithCredential` or `Authenticate` is more appropriate.
``` C#
var serviceManager = new ServiceManagerBuilder()
                    .Credentials(connectionString) 
                    // .Credentials(endpoint, accessToken)
                    // .Credentials(endpoint, accessTokenList)
                    .WithOptions(option => 
                    {
                        option.SeriveHubContextBackend = SeriveHubContextBackend.Rest
                        // option.SeriveHubContextBackend = SeriveHubContextBackend.Websocket
                    })
                    .Build();
```

Reply:
I mean WithCredentials...

Reply:
Got it, thanks...